server/lib/validation/languages/tok.js

```diff
@@ -0,0 +1,70 @@
+const tokenizeWords = require('talisman/tokenizers/words');
+
+const TRANSLATION_KEY_PREFIX = 'TRANSLATION_KEY:';
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 18;
+
+const INVALIDATIONS = [{
+  fn: (sentence) => {
+    const words = tokenizeWords(sentence);
+    return words.length < MIN_WORDS || words.length > MAX_WORDS;
+  },
+  error: `${TRANSLATION_KEY_PREFIX}sc-validation-number-of-words`,
```

Comment:
I think we could replace this error translation string with the actual translation. The default file has these to make sure that the default fallback error messages are translated. When moving these out to a dedicated validation file by language I'd say it would be better to have the translations here. Otherwise if we ever change something in the default validator the error message might not be correct anymore. The current translations can be found here: https://pontoon.mozilla.org/tok/common-voice/web/locales/en/messages.ftl/?search=sc-validation&string=235972

This also applies to the other 2 entries right below.

Reply:
Thanks for the pointer. I updated accordingly.